tests/build/depfile.srctree

```diff
@@ -0,0 +1,36 @@
+CYTHONIZE -M foo.pyx
+PYTHON -c'import check.py'
+
+######## foo.pyx ########
+
+from bar cimport empty
+
+include "baz.pxi"
+
+empty()
+print(foo())
+
+
+######## baz.pxi ########
+
+def foo():
+    return "foo"
+
+
+
+######## bar.pxd ########
+
+cdef inline void empty():
+    print("empty")
+
+
+
+
+######## check.py ########
+
+import os
+
+with open("foo.c.dep", "r") as f:
+    contents = f.read().replace("\n", " ").replace("\\", "")
+
+assert sorted(contents.split()) == ['bar.pxd', 'baz.pxi', 'foo.c:', 'foo.pyx']
```

Comment:
We should definitely also test the behaviour for packages and non-package subdirectories. Probably also dependencies given with absolute paths, e.g. header files installed elsewhere such as NumPy's include files (which would best go into a separate test file and require a `# tag: numpy`.)